digital_land/phase/lookup.py

```diff
@@ -24,12 +24,24 @@ class LookupPhase(Phase):
     lookup entity numbers by CURIE
     """
 
-    def __init__(self, lookups={}):
+    entity_field = None
+
+    def __init__(self, lookups={}, redirect_lookups={}, issue_log=None):
         self.lookups = lookups
+        self.redirect_lookups = redirect_lookups
+        self.issues = issue_log
 
     def lookup(self, **kwargs):
         return self.lookups.get(key(**kwargs), "")
 
+    def get_entity_number(self, entity):
```

Comment:
this name is a bit misleading, the function is checking for a redirect and returning the new entity number if it it (or none if it's been retired).

you could expand the function even more and include getting the number from lookups as well then re use it across both classes rather than needing to use super